src/main/resources/db/migration/V1__init.sql

```diff
@@ -1,13 +1,3 @@
-CREATE TABLE IF NOT EXISTS foodservice_test
```

Comment:
V2 script banao if you want to update or add new tables.. this will result in failure as checksum mismatch will happen